test/e2e/serial/tests/workload_overhead.go

```diff
@@ -243,9 +251,14 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload overhea
 					_, err = e2enrt.CheckZoneConsumedResourcesAtLeast(*nrtInitial, *nrtPostCreate, podResources)
```

Comment:
don't we need to check that the output value (here is ignored "_") is not empty because of the same reason that we check it is empty in line 254?

Reply:
yep, I should have fixed also this. Thanks!

Reply:
we do, revisited the checks in the last upload